### PR TITLE
project init

### PR DESCRIPTION
**Pull request by Catalin12**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
+indent_size = 3
```
Comment (danielursu) [inline .editorconfig]:
```suggestion
indent_size = 4
```

Comment (carmina-c) [inline .editorconfig]: indent_size = 4

File: src/main.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AppModule } from './app/app.module';
```
Comment (danielursu) [inline src/main.ts]: double quotes

File: src/polyfills.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ *  with the following flag, it will bypass `zone.js` patch for IE/Edge
+ *
+ *  (window as any).__Zone_enable_cross_context_check = true;
+ *
+ */
+
+/***************************************************************************************************
+ * Zone JS is required by default for Angular itself.
+ */
+import 'zone.js';  // Included with Angular CLI.
```
Comment (danielursu) [inline src/polyfills.ts]: double quotes

File: src/test.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// This file is required by karma.conf.js and loads recursively all the .spec and framework files
+
+import 'zone.js/testing';
+import { getTestBed } from '@angular/core/testing';
```
Comment (danielursu) [inline src/test.ts]: double quotes

**Review by danielursu — CHANGES_REQUESTED** [review]

File: karma.conf.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        { type: 'html' },
+        { type: 'text-summary' }
+      ]
+    },
+    reporters: ['progress', 'kjhtml'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: true,
+    browsers: ['Chrome'],
```
Comment (carmina-c) [inline karma.conf.js]: double quotes here too?

File: src/main.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+import { environment } from './environments/environment';
+
+if (environment.production) {
+	enableProdMode();
+}
+
+platformBrowserDynamic().bootstrapModule(AppModule)
+	.catch(err => console.error(err));
```
Comment (carmina-c) [inline src/main.ts:12]: same here, double quotes

**Review by danielursu — APPROVED** [review]

**Review by carmina-c — APPROVED** [review]

**Pull request merged**